lib/index.js

```diff
@@ -20,6 +20,8 @@ var Storage = require('./util/storage');
 var promptSuggestion = require('./util/prompt-suggestion');
 var debug = require('debug')('yeoman:generator');
 
+var EMPTY = '@@_YEOMAN_EMPTY_MARKER_@@';
```

Comment:
I need a way to detect if an explicit value was passed or not. I though of using a `Symbol` given it'd be a guaranteed unique value, but minimist breaks if `default` is not a string or a number.

So, I went with a long string that hopefully won't collide with any real value